packages/flutter/lib/src/material/outlined_button.dart

```diff
@@ -7,6 +7,7 @@ import 'dart:ui' show lerpDouble;
 
 import 'package:flutter/foundation.dart';
 import 'package:flutter/rendering.dart';
+import 'package:flutter/src/material/button.dart';
```

Comment:
> For old buttons(FlatButton, RaisedButton, and OutlineButton) in flutter/packages/flutter/lib/src/material/button.dart. 
> For new buttons(TextButton, ElevatedButton, and OutlinedButton) in `flutter/packages/flutter/lib/src/material/button_style_button.dart`.

As per my comment on the issue, this shouldn't be included here. You were probably trying to import this in **OutlineButton**.

I'm kind of conflicted, because doing as quoted above would result in code duplication.
Someone having a better experience with material buttons may be able to guide here.

Reply:
I believe the enum is in the best place right now